source/deployment/enterprise-deployment-guide.rst

```diff
@@ -0,0 +1,216 @@
+
+
+.. Note::
+    Reach out to our Sales team by contacting sales@mattermost.com to upgrade your license at any time.
+
+Mattermost Server Environment
+-----------------------------
+
+At its core, Mattermost is an open source, hybrid-cloud alternative to proprietary SaaS messaging for teams. It is designed to increase the agility, efficiency and innovation in high trust organizations while keeping data and operations under IT control. Mattermost ensures you can own your data. An important step is to determine the environment for the Mattermost server.
+
+We officially support running the Mattermost Server on multiple `Linux distributions, Docker and on-premises cloud solutions <https://docs.mattermost.com/guides/administrator.html#installing-mattermost>`_.
+
+.. Note::
+    Other Linux distributions might work too, but are not officially supported.
+
+Client Usage
+------------
+
+Depending on your environment and your users, you have several options when deploying the Mattermost Client: Web Browser, `Desktop <https://docs.mattermost.com/install/desktop.html>`_, `Mobile <https://docs.mattermost.com/mobile/mobile-overview.html>`_ or all three. We `provide binaries <https://mattermost.com/download/#mattermostApps>`_ for MS Windows, macOS, Linux, iOS and Android. However, depending on your organization policies deployment on these platforms can vary.
+
+======================
+Deployment First Steps
+======================
+
+Deploying Mattermost Enterprise is not a small project. Depending on your use-case multiple physical machines have to be set up with Mattermost server, a proxy and a database while thousands of users need to be imported through AD/FS. While we strive to make this as easy as possible it will take time and iteration.
+
+***********************
+Determine Your Use-Case
+***********************
+
+As mentioned above it is essential for a successful deployment to know your specific use-case. To get started try answering the following questions:
+
+- How many users will use Mattermost on initial deployment and is this number going to increase dramatically in the near future?
+- What clients will be in use?
+- Are you migrating from an existing ChatOps or different communications platform?
+- Are you using an identity provider for single sign-on and if yes which one?
+- What compliance requirements do you need to meet?
+- What are your organization's security requirements?
+
+************************
+Planning Your Deployment
+************************
+
+Technical Requirements
+----------------------
+
+The hardware requirements for the Mattermost server and database `grow with the amount of users they need to support <https://jeffschering.github.io/mmdocs/upgrade/install/requirements.html#hardware-requirements>`_.
```

Comment:
Is this link correct?  Recommend replacing with our link: https://docs.mattermost.com/install/requirements.html#hardware-requirements